js_modules/dagit/packages/core/src/schedules/SchedulesTable.tsx

```diff
@@ -131,14 +131,8 @@ const ScheduleRow: React.FC<{
   const repo = useRepository(repoAddress);
   const isJob = isThisThingAJob(repo, schedule.pipelineName);
 
-  const {
-    name,
-    cronSchedule,
-    executionTimezone,
-    futureTicks,
-    pipelineName,
-    scheduleState,
-  } = schedule;
+  const {name, cronSchedule, executionTimezone, futureTicks, pipelineName, scheduleState} =
+    schedule;
```

Comment:
This is a bit of a weird layout of this - new prettier?

Reply:
Yeah. I don't know why they did this.